7_react/react1/week2/src/TodoList/TodoList.js

```diff
@@ -0,0 +1,53 @@
+import TodoItem from "TodoItem/TodoItem"
+import "./TodoList.css"
+
+
+const todoList = [{
+    num: 1,
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017"
+},
+{
+    num: 2,
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017"
+},
+{
+    num: 3,
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017"
+}]
+
+function TodoList(){
+    const todoItemsComponents = todoList.map(item => (
+        <TodoItem key={item.num} description={item.description} deadline={item.deadline}/>
+    ))
+        function App() {
```

Comment:
To my knowledge, you should never embed a component into another one. I.e., App and TodoList should be defined separately. (Here App is defined inside TodoList)